client/demo/businessData/hv-messages-and-questions

```diff
@@ -0,0 +1,93 @@
+{
+  "possibleRecipients": [
+    {
+      "id": "ENTITY1_FR"
+    },
+    {
+      "id": "ENTITY2_FR"
+    },
+    {
+      "id": "ENTITY3_FR"
+    },
+    {
+      "id": "ENTITY4_FR"
+    },
+    {
+      "id": "ENTITY1_IT"
+    },
+    {
+      "id": "ENTITY2_IT"
+    },
+    {
+      "id": "ENTITY3_IT"
+    },
+    {
+      "id": "ENTITY1_NL"
+    },
+    {
+      "id": "ENTITY2_NL"
+    },
+    {
+      "id": "ENTITY1_EU"
+    },
+    {
+      "id": "ENTITY2_EU"
+    }
+  ],
+  "messagesList": [
+    {
+        "id": "VoltageReduction",
+        "title": "Coordinated Voltage Reduction Plan",
+        "richMessage":{"ops":[{"insert":"In anticipation of the low volume of transit at interconnections and the consumption trough, please implement all normal means to lower voltages."},{"attributes":{"header":2},"insert":"\n\n"},{"insert":"\n"}]},
+        "severity": "ACTION",
+        "question": false,
+        "recipients": [
+          "ENTITY1_FR",
+          "ENTITY2_FR",
+          "ENTITY4_FR"
+        ],
+        "recipientsForInformation": []
+    },
+    {
+      "id": "Maintenance",
+      "title": "Urgent maintenance scheduled",
+      "richMessage": {"ops":[{"insert":"Maintenance work for HV line "},{"attributes":{"color":"#e60000"},"insert":"XXX"},{"insert":"  is shedule today between  "},{"attributes":{"color":"#e60000"},"insert":"XX:XX AM/PM"},{"insert":" and "},{"attributes":{"color":"#e60000","bold":true},"insert":"YY:YY: AM/PM"},{"attributes":{"bold":true},"insert":" "},{"attributes":{"header":2},"insert":"\n\n"},{"insert":"\n"}]},
```

Comment:
change 
{"insert":" is shedule today between "}
into
{"insert":" is scheduled today between "}